osu.Android/GameplayScreenRotationLocker.cs

```diff
@@ -6,28 +6,29 @@
 using osu.Framework.Bindables;
 using osu.Framework.Graphics;
 using osu.Game;
+using osu.Game.Screens.Play;
 
 namespace osu.Android
 {
     public partial class GameplayScreenRotationLocker : Component
     {
-        private Bindable<bool> localUserPlaying = null!;
+        private IBindable<LocalUserPlayingStates> localUserPlaying = null!;
```

Comment:
What happened here? The class doesn't exist?

Reply:
Probably something something missed in the rename because `osu.Desktop.slnf`.

```suggestion
        private Bindable<bool> localUserPlaying = null!;
        private IBindable<LocalUserPlayingState> localUserPlaying = null!;
```

probably fixes.